style.css

```diff
@@ -777,25 +816,349 @@ body {
   margin-left: 35px;
 }
 
+.menutab {
+  display: flex;
+  align-items: flex-end;
+  gap: 20px;
+}
+
 .sectioncontact {
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  padding-bottom: 35px;
 }
 
-#Name {
-  padding-top: 15px;
-  padding-bottom: 15px;
-  padding-left: 16px;
-  margin-bottom: 10px;
-  border-radius: 8px;
-}
-
-#email {
-  padding-top: 15px;
-  padding-bottom: 15px;
-  padding-left: 16px;
-  margin-bottom: 10px;
-  border-radius: 8px;
+.Portfolio-desc,
+.About-desc,
+.Programs-desck,
+.Programs-desclist,
+.Leaddev,
+.uber,
+.contact-desc,
+.card_02_desc,
+.cardbackid03,
+.card_05_desck,
+.card_03_desck,
+.card_04-desck,
+.facebook,
+.canopy-desck,
+.Ruby,
+.fullstack,
+.section2,
+.section1-desc,
+.Multi-Post_05-desck,
+.Facebook360,
+.button2 {
+  display: none;
+}
+
+.Programs-desck {
+  display: none;
+}
+
+@media screen and (min-width: 376px) {
```

Comment:
Kindly use media query min-width: 768 to the desktop view. Right now, you have no breakpoints for the desktop. And please fix the overflow issue occurring on mobile view. Please check the gif below.
![mobile](https://user-images.githubusercontent.com/25887253/208170840-6c0885a2-bf4b-41e6-87ec-a12aaff8071c.gif)